gravity-forms/gw-update-posts.php

```diff
@@ -331,4 +412,4 @@ public function get_post_date( $entry, $form ) {
 		return gmdate( 'Y-m-d H:i:s', strtotime( sprintf( '%s %s:%s:00', $date, $hour, $min ) ) );
 	}
 
-}
+}
```

Comment:
We typically leave a newline at the end of files. Mostly to help ensure compat with certain older tools and it's also a bit better in case something gets concatenated at the end of the file.

```suggestion
}

```

Reply:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file